integreatly-mobile-security-service/0.4.1/mobile-security-service-operator.v0.4.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,184 @@
+                - update
+                - patch
+                - delete
+            - apiGroups:
+                - batch
+              resources:
+                - cronjobs
+              verbs:
+                - get
+                - list
+                - watch
+                - create
+                - update
+                - patch
+                - delete
+            - apiGroups:
+                - route.openshift.io
+              resources:
+                - routes
+              verbs:
+                - get
+                - list
+                - watch
+                - create
+                - update
+                - patch
+                - delete
+            - apiGroups:
+                - apps
+              resources:
+                - deployments
+              verbs:
+                - get
+                - list
+                - watch
+                - create
+                - update
+                - patch
+                - delete
+            - apiGroups:
+                - ""
+              resources:
+                - pods
+              verbs:
+                - get
+                - list
+                - watch
+            - apiGroups:
+                - apps
+              resources:
+                - replicasets
+              verbs:
+                - get
+            - apiGroups:
+                - apps
+              resourceNames:
+                - mobile-security-service-operator
+              resources:
+                - deployments/finalizers
+              verbs:
+                - update
+            - apiGroups:
+                - ""
+              resources:
+                - namespaces
+              verbs:
+                - get
+          serviceAccountName: mobile-security-service-operator
+      deployments:
+        - name: mobile-security-service-operator
+          spec:
+            replicas: 1
+            selector:
+              matchLabels:
+                name: mobile-security-service-operator
+            strategy: {}
+            template:
+              metadata:
+                labels:
+                  name: mobile-security-service-operator
+              spec:
+                containers:
+                  - command:
+                      - mobile-security-service-operator
+                    env:
+                      - name: WATCH_NAMESPACE
+                        valueFrom:
+                          fieldRef:
+                            fieldPath: metadata.namespace
+                      - name: APP_NAMESPACES
+                        value: mobile-security-service-apps
+                      - name: POD_NAME
+                        valueFrom:
+                          fieldRef:
+                            fieldPath: metadata.name
+                      - name: OPERATOR_NAME
+                        value: mobile-security-service-operator
+                    image: quay.io/aerogear/mobile-security-service-operator:master
```

Comment:
it's better to point at a specific tag, as `master` could change in the future and break our install without us doing anything.